README.md

```diff
@@ -31,76 +31,53 @@ Execute the following commands to create a duplicate of a repository; you need t
-    Requested Format ID (ex: Onshape-Demo/BOM)
+Onshape will register the app on Partner server and send back the OAUTH Secret which are required for the next step. SAVE IT! It will not be visible again unless you reset the key for security reasons.
 
-Onshape will register the app on Partner server and send back the OAUTH ID/Secret which are required for authentication.
+#### **Set Heroku App Environment**
 
-Make changes to code at two places for the new URL that Heroku has produced, as shown below:
+Make changes to code at one place to point the app to the URL Heroku has created:
 
-    file# 1: ./package.json
+    file: ./package.json
 
        .........
        ........
        "repository": {
        "type": "git",
-       "url": "https://newURL-from-heroku.herokuapp.com/"
+       "url": "<MY_APP_URL>"
        },
        ...........
 
-   And
-
-    file# 2: ./authentication.js
-
-        ...........
-       passport.use(new OnshapeStrategy({
-         clientID: oauthClientId,
-         clientSecret: oauthClientSecret,
-         callbackURL: "https://newURL-from-heroku.herokuapp.com/oauthRedirect",
-         .............
-       },
-       function(accessToken, refreshToken, profile, done) {
-         ...........
-
 Push the local repo code along with code changes to heruko
 
     $ git add package.json
-    $ git add authentication.js
     $ git commit -am "changes to code for callbackURL"
-
     $ git push heroku master
 
-You will need to set the ID and Secret as environment variables on the server. These are only visible to the app running on the server preserving security of that information.
+You will need to set the ID and Secret as environment variables on the server. These are only visible to the app running on the server, which preserves the security of that information.
 
-    $ heroku config:set OAUTH_CLIENT_ID=<ID given by Onshape for this app>
-    $ heroku config:set OAUTH_CLIENT_SECRET=<Secret given by Onshape for this app>
-    $ heroku config:set OAUTH_CALLBACK_URL=<https://newURL-from-heroku.herokuapp.com>
-
-You will also need to register your server host, the stack url (for example ONSHAPE_PLATFORM=https://cad.onshape.com), and the Onshape authentication URL.
-
-    $ heroku config:set ONSHAPE_HOST=https://newURL-from-heroku.herokuapp.com
-    $ heroku config:set ONSHAPE_PLATFORM=https://STACK.onshape.com
+    $ heroku config:set OAUTH_CLIENT_ID=<ID given by Onshape for this app - available in the dev-portal. Include the `=`'s on the end>
+    $ heroku config:set OAUTH_CLIENT_SECRET=<Secret given by Onshape for this app when you make the OAuth app.  Include the `=`'s on the end>
+    $ heroku config:set OAUTH_CALLBACK_URL=<MY_APP_URL>/oauthRedirect
+    $ heroku config:set ONSHAPE_HOST=<MY_APP_URL>
+    $ heroku config:set ONSHAPE_URL=https://cad.onshape.com
     $ heroku config:set ONSHAPE_OAUTH_SERVICE=https://oauth.onshape.com
+    $ heroku config:set API_URL=https://staging.dev.onshape.com
```

Comment:
This is incorrect - the `API_URL` should match the `ONSHAPE_URL` - it should be `https://cad.onshape.com`

Given that API_URL and ONSHAPE_URL match the sample should really be updated to remove the API_URL references and replace them with ONSHAPE_URL.